snapshot/snapshot.js

```diff
@@ -117,6 +120,10 @@ const fetchAssets = async (contractAddress, cursor, retry) => {
 
 const fetchAssetsOfRange = async (contractAddress, range, retry) => {
   let toSnapshot = [];
+  console.log("fetchAssetsOfRange", { contractAddress, range, retry });
+  if (range[0] > range[1]) {
+    return toSnapshot;
+  }
```

Comment:
handles an edge case – if the token ID is a perfect multiple of 30, the script would run forever!